### PR TITLE
docs: minor cli reference editorial updates

### PR DESCRIPTION
**Pull request by dvdksn**
editorial and formatting updates

### CODE REVIEW

File: docs/reference/buildx_version.md [hunk 1]
```diff
@@ -16,5 +16,5 @@ View version information
 
 ```console
 $ docker buildx version
-github.com/docker/buildx v0.5.1-docker 11057da37336192bfc57d81e02359ba7ba848e4a
+github.com/docker/buildx v0.11.0-263-gd537b9e4.m d537b9e418b909b65274a902e25b3d1757553a3b.m
```
Comment (crazy-max) [inline docs/reference/buildx_version.md:19]: Maybe show a GA release?:

```
github.com/docker/buildx v0.11.2 9872040b6626fb7d87ef7296fd5b832e8cc2ad17
```

File: docs/reference/buildx_imagetools_inspect.md [hunk 2]
```diff
@@ -123,13 +123,11 @@ Manifests:
...
 
-A `json` go template func is also available if you want to render fields as
-JSON bytes:
+A `json` template function is also available if you want to render fields in
+JSON format:
 
 ```console
 $ docker buildx imagetools inspect crazymax/loop --format "{{json .Manifest}}"
-```
-```json
```
Comment (crazy-max) [inline docs/reference/buildx_imagetools_inspect.md:132]: I don't have a really strong preference but I like to highlight JSON output

image

File: docs/reference/buildx_create.md [hunk 3]
```diff
@@ -135,59 +143,18 @@ to achieve that.
...
-#### `remote` driver
-
-- `key=KEY` - Sets the TLS client key.
-- `cert=CERT` - Sets the TLS client certificate to present to buildkitd.
-- `cacert=CACERT` - Sets the TLS certificate authority used for validation.
-- `servername=SERVER` - Sets the TLS server name to be used in requests (defaults to the endpoint hostname).
+* [`docker` driver](https://docs.docker.com/build/drivers/docker/)
+* [`docker-container` driver](https://docs.docker.com/build/drivers/docker-container/)
+* [`kubernetes` driver](https://docs.docker.com/build/drivers/kubernetes/)
+* [`remote` driver](https://docs.docker.com/build/drivers/remote/)
```
Comment (crazy-max) [inline docs/reference/buildx_create.md:157]: I agree that we have these opts in multiple places. We should also make sure docs is being updated when changes are being made. Maybe we should add a comment like in https://github.com/docker/docs/blob/main/content/build/drivers/docker-container.md:

```go
type Driver struct {
	driver.InitConfig
	factory      driver.Factory

    // if you add fields, do not forget to update https://github.com/docker/docs/blob/main/content/build/drivers/docker-container.md
	netMode      string
	image        string
	memory       opts.MemBytes
	memorySwap   opts.MemSwapBytes
	cpuQuota     int64
	cpuPeriod    int64
	cpuShares    int64
	cpusetCpus   string
	cpusetMems   string
	cgroupParent string
	env          []string
}
```

**Review by crazy-max — CHANGES_REQUESTED** [review]
Some lint issues otherwise LGTM: https://github.com/docker/buildx/actions/runs/6892599910/job/18750273469?pr=2117#step:4:106

```
#13 83.37 driver/kubernetes/driver.go:41: File is not `gofmt`-ed with `-s` (gofmt)
#13 83.37 	factory          driver.Factory
#13 83.37 driver/docker-container/driver.go:43: File is not `gofmt`-ed with `-s` (gofmt)
#13 83.37 	factory      driver.Factory
#13 ERROR: process "/bin/sh -c golangci-lint run" did not complete successfully: exit code: 1
```

**Review by crazy-max — APPROVED** [review]
LGTM

**Review by tonistiigi — APPROVED** [review]

**Pull request merged**